app/src/main/java/in/testpress/testpress/authenticator/LoginActivity.java

```diff
@@ -522,6 +530,28 @@ protected void finishLogin() {
         finish();
     }
 
+    private void deleteDownloadedVideos() {
+        if (isUserLoginWithDifferentAccount()) {
+            Executor executor = Executors.newSingleThreadExecutor();
+            executor.execute(() -> TestpressDatabase.Companion.invoke(this).clearAllTables());
+            DownloadService.sendRemoveAllDownloads(this, VideoDownloadService.class, false);
+            TestpressSDKDatabase.clearDatabase(this);
+        }
```

Comment:
This method says deleteDownloadedVideos but it is  checking something and then deleting